cg/server/endpoints/orders.py

```diff
@@ -263,18 +266,24 @@ def get_options():
     )
 
 
-@ORDERS_BLUEPRINT.route("/validate_order/<workflow>", methods=["POST"])
-def validate_order(workflow: str):
+@ORDERS_BLUEPRINT.route("/validate_order/<order_type>", methods=["POST"])
+def validate_order(order_type: OrderType):
     raw_order = request.get_json()
-    raw_order["workflow"] = workflow
+    raw_order["workflow"] = order_type
     raw_order["user_id"] = g.current_user.id
     response = {}
-    if workflow == Workflow.MICROSALT:
+    if order_type == OrderType.BALSAMIC:
```

Comment:
I will create a little work item for refactoring this endpoint - we should have a map between order types and validation services or something similar. I would deem it out of scope for this PR though